app/lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+class Menu
```

Comment:
non blocking: I would add a line or two of documentation as it's not self explanatory what this class is for unless you find `AdminMenu` first

Reply:
I just added a one line now to describe it, I may look at this again and expand on it. ✅